example/scalalib/native/3-multi-module/build.mill

```diff
@@ -0,0 +1,91 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+trait MyModule extends ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+
+  def nativeLinkingOptions = Seq("-L" + millSourcePath.toString + "/target")
+}
+
+object foo extends MyModule {
+  def moduleDeps = Seq(bar)
+
+  def ivyDeps = Agg(ivy"com.lihaoyi::mainargs::0.7.6")
+
+  // Compile C
+  def nativeCompiled = T {
+    os.makeDir.all(millSourcePath / "target")
+
+    os.proc(
```

Comment:
This `os.proc` call should be moved into `MyModule`

Reply:
I'm a little bit confused here. I'm writing the multi-module example for `foo` to depends on `bar`.
Which os.proc shoiuld be in `MyModule` based on that?

How I understand from reading the doc is that since both `foo` and `bar`are to extends from `MyModule` and that `foo` depends on `bar` so  `bar` os.proc should be move to `MyModule`

But if you think otherwise please let me know